### PR TITLE
`Style/TrailingCommaInArguments`, `Style/TrailingCommaInArrayLiteral`, and `Style/TrailingCommaInHashLiteral`

### PR DESCRIPTION
**Pull request by kbarrette**
Configure `Style/TrailingCommaInArrayLiteral` and `Style/TrailingCommaInHashLiteral` to require consistent commas in multiline literals

### CODE REVIEW

File: conf/rubocop_without_rspec.yml [hunk 1]
```diff
@@ -231,3 +231,9 @@ Style/RedundantSelfAssignment:
 
 Style/SingleArgumentDig:
   Enabled: false
+
+Style/TrailingCommaInArrayLiteral:
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/TrailingCommaInHashLiteral:
```
Comment (erikkessler1) [inline conf/rubocop_without_rspec.yml:241]: Should we include `Style/TrailingCommaInArguments` as well?

Comment (ermanc-salsify) [reply conf/rubocop_without_rspec.yml:241]: I'm up for it. Rules and patterns seems to be the same, it also has safe auto-correct so 👍

Comment (kbarrette) [reply conf/rubocop_without_rspec.yml:241]: 👍 I added a commit here for `Style/TrailingCommaInArguments`

File: lib/rubocop/cop/salsify/style_dig.rb [hunk 2]
```diff
@@ -50,7 +50,7 @@ def autocorrect(node)
           lambda do |corrector|
             range = Parser::Source::Range.new(node.source_range.source_buffer,
                                               root_node.source_range.end_pos,
-                                              node.source_range.end_pos)
+                                              node.source_range.end_pos,)
```
Comment (ermanc-salsify) [inline lib/rubocop/cop/salsify/style_dig.rb]: It's ironic that the major benefit of trailing-commas is hard to realize when it's used with same-line-parens (or brackets). Oh well, better than before...

**Review by ermanc-salsify — APPROVED** [review]

File: lib/rubocop/cop/salsify/style_dig.rb (hunk 2 above)
Comment (erikkessler1) [reply lib/rubocop/cop/salsify/style_dig.rb]: This is what makes me think we should switch from `consistent_comma` to `comma`.

File: conf/rubocop_without_rspec.yml (hunk 1 above)
Comment (jturkel) [reply conf/rubocop_without_rspec.yml:241]: We should bring the `Style/TrailingCommaInArguments` cop question back to #rubocop-changes for a broader discussion

**Review by erikkessler1 — APPROVED** [review]

**Comment by kbarrette** [comment]
Going to leave this as-is for now - these cops are enabled by default with `no_comma`, and we don't have consensus on whether it's worth changing this vs the potential number of violations to fix.

**Pull request closed without merging**